.pipelines/e2e.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       export BUILD_ID=$(Build.BuildId)
 
       cd e2e
-      go test -timeout ${{ variables.defaultTimeout }}m -v -run Test_All ./
+      go test -parallel 100 -timeout ${{ variables.defaultTimeout }}m -v -run Test_All ./
```

Comment:
could we expose  the parallel count as an external pipeline variable to allow changing outside of the code in case it causes issues in the future

Reply:
added